meta/main.yml

```diff
@@ -5,21 +5,18 @@ galaxy_info:
   description: Install and configure the PowerDNS Recursor
   company: PowerDNS.COM BV
   license: MIT
-  min_ansible_version: 2.2
+  min_ansible_version: 2.7
   platforms:
     - name: EL
       versions:
         - 7
         - 6
```

Comment:
I see centos-6 was removed from molecule platform list, should it be also remove from this list and added 8? I guess this was the intention.